docs/daemon/configuration/configuration.mdx

```diff
@@ -0,0 +1,139 @@
+
+## chain:
+This section contains configuration specific to cometbft based blockchain.
+
+- `moniker`: This variable sets a custom human-readable name for the node.
+
+### rpc
+This subsection contains the configuration required to manage blockchain's rpc service.
+
+- `laddr`: This variable sets the TCP or UNIX socket address for the RPC server to listen on. Default listening address is on "tcp://127.0.0.1:26657"
+
+- `timeout_broadcast_tx_commit`: This variable sets the time to wait for a transaction to be committed during */broadcast_tx_commit*. Default timeout is 10s.
+
+### consensus
+This subsection describes the configuration to manage consensus.
+
+- `timeout_propose`: This variable sets the time to wait for a proposal block before prevoting nil. Default timeout is 3s.
+
+- `timeout_prevote`: This variable sets the time to wait after receiving +2/3 prevotes for "anything". Default timeout is 1s.
+
+- `timeout_precommit`: This variable sets the time to wait after receiving +2/3 precommits for "anything". Default timeout is 1s.
+
+- `timeout_commit`: This variable sets the time to wait after committing a block before starting on the new height. Default timeout is 1s.
+
+### p2p
+This subsection describes the configuration to manage the p2p service handling peer connectivity between the nodes in the blockchain network.
+
+- `laddr`: This variable sets the Address to listen for incoming connections. Default listening address is "tcp://0.0.0.0:26656".
+
+- `external_address`: This variable sets the Address to advertise to peers for them to dial. If empty, will use the same port as the laddr, and will introspect on the listener.
+
+- `persistent_peers`: This variable sets the Comma-separated list of nodes to connect to and maintain persistent connections. This is a required field for a new node to connect to the kwild network.
+
+- `addr_book_strict`: Toggle for strict address routability rules.
+
+- `max_num_inbound_peers`: This variable sets the Maximum number of inbound peers.
+
+- `max_num_outbound_peers`: This variable sets the Maximum number of outbound peers to connect to, excluding persistent peers.
+
+- `unconditional_peer_ids`: This variable sets the List of node IDs for which connections will be (re)established irrespective of the max connections.
+
+- `allow_duplicate_ip`: This variable sets the Toggle to disable guard against peers connecting from the same IP.
+
+### mempool
+This subsection describes the configuration to manage blockchain's transaction mempool.
+
+- `size`: This variable sets the Maximum number of transactions in the mempool.
+
+- `max_txs_bytes`: This variable sets the Limit on the total size of all transactions in the mempool.
+
+- `cache_size`: This variable sets the Size of the cache used to filter transactions seen earlier.
+
+This is the structure in which these configs must appear in the config.toml file. Here is a sample [config.toml](./default_config.toml) for reference. 
```

Comment:
I think it's weird that it downloads a config file to your computer.  Can you just show using:

```yaml
key: value
```

Reply:
Just realized I made it yaml, but point still stands